src/main/java/com/google/api/codegen/InterfaceListView.java

```diff
@@ -18,26 +18,25 @@
 import com.google.api.tools.framework.model.Model;
 
 import java.util.ArrayList;
-import java.util.List;
 
 /**
  * An interface-based view of model, consisting of a strategy for getting the interfaces of the
  * model and returning them as a single list object.
  */
-public class InterfaceListView implements InputElementView<List<Interface>> {
+public class InterfaceListView implements InputElementView<Iterable<Interface>> {
 
   /**
    * Gets the reachable interfaces of the model.
    */
   @Override
-  public Iterable<List<Interface>> getElementIterable(Model model) {
+  public Iterable<Iterable<Interface>> getElementIterable(Model model) {
```

Comment:
This can be done separately:
`return Collections.singleton(new InterfaceView().getElementIterable(model));` ?